src/Blt/Plugin/Commands/SwsCommands.php

```diff
@@ -230,4 +232,126 @@ protected function getSiteAliases() {
+          continue;
+        }
+      }
+
+      $this->say(sprintf('Creating release for %s', $package_name));
+      $this->createRelease($package);
+    }
+  }
+
+  /**
+   * Edit the Changelog and create a pull request for a package.
+   *
+   * @param string $dir
+   */
+  protected function createRelease($info_path) {
+    $dir = dirname($info_path);
+    $base_branch = 'master';
+    $branch_name = $this->taskGit()
+      ->dir($dir)
+      ->exec('rev-parse --abbrev-ref HEAD')
+      ->printOutput(FALSE)
+      ->run()
+      ->getMessage();
+    if ($branch_name == 'HEAD') {
+      throw new \Exception('Invalid branch name for ' . $dir);
+    }
+
+    $this->taskGit()->dir($dir)->pull('origin', $branch_name)->run();
+
+    $info = Yaml::decode(file_get_contents($info_path));
+    $version = str_replace('-dev', '', $info['version']);
+    $anchor = fgets(fopen("$dir/CHANGELOG.md", 'r'));
+
+    $log = $this->taskGit()
+      ->dir($dir)
+      ->exec("log --oneline origin/$base_branch..HEAD")
+      ->printOutput(FALSE)
+      ->run()
+      ->getMessage();
+
+    $log = explode("\n", $log);
+    foreach ($log as $key => &$line) {
+      $line = preg_replace('/^.*? /', '- ', $line);
+      if (strpos(strtolower($line), 'back to dev') !== FALSE) {
+        unset($log[$key]);
+      }
+    }
+
+    if (empty($log)) {
+      return;
+    }
+    $header = [
+      $version,
+      str_repeat('-', 80),
+      '_Release Date: ' . date('Y-m-d') . '_',
+      PHP_EOL,
+    ];
+
+    $this->taskChangelog("$dir/CHANGELOG.md")
+      ->version($version)
+      ->setHeader(implode(PHP_EOL, $header))
+      ->setBody(implode("\n", $log) . PHP_EOL)
+      ->anchor($anchor)
+      ->run();
+    $composer_diff = $this->taskGit()
+      ->dir($dir)
+      ->exec("diff origin/$base_branch..HEAD -- composer.json")
+      ->printOutput(FALSE)
+      ->run()
+      ->getMessage();
+
+    $matches = preg_grep('/^-.*su-.*?$/', explode("\n", $composer_diff));
+    foreach ($matches as $match) {
+      preg_match('/(su-[a-z]+\/[a-z_-]+)/', $match, $matched_packages);
+      $this->taskComposerRequire()
+        ->dir($dir)
+        ->arg($matched_packages[1])
+        ->option('no-update')
+        ->run();
+    }
+
+    $finder = new Finder();
+    $finder->in($dir)
+      ->files()
+      ->name('*.info.yml');
+
+    foreach ($finder as $dir) {
+      $info_file = $dir->getRealPath();
+      $contents = file_get_contents($info_file);
+      file_put_contents($info_file, preg_replace('/version: (.*)-dev/', 'version: $1', $contents));
+    }
+
+    $release_version = str_replace('x-', '', $version);
+    $this->taskGit()
+      ->dir($dir)
+      ->checkout("-b release-$release_version")
+      ->add($dir)
+      ->commit($release_version)
+      ->push('origin', "release-$release_version")
+      ->run();
+    sleep(1);
+    $this->taskExec("gh pr create -B $base_branch --title $release_version --body $version")
```

Comment:
Worth noting that it requires `gh` be installed, which is likely not the case for everyone. 🤷